spec/takeaway_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'takeaway'
+
+describe Takeaway do
+  let(:order_klass) {
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)